packages/lib-classifier/src/store/TutorialStore/TutorialStore.js

```diff
@@ -30,12 +30,13 @@ const TutorialStore = types
       return self.loadingState !== asyncStates.success || (self.loadingState === asyncStates.success && !self.tutorial)
     },
 
-    get stepWithMedium () {
-      if (self.isActiveStepValid) {
-        const step = self.active.steps[self.activeStep]
-        return { step, medium: self.activeMedium }
+    stepWithMedium(index) {
+      const step = self.active.steps[index]
```

Comment:
```suggestion
      const step = self.active?.steps[index]
```

ModalTutorial "Get Started" works, but SlideTutorial "Get Started" errors on this line trying to read `steps` with `self.active = undefined`.

Reply:
I think `self.active` here should also be wrapped in `tryReference()`. I'll double check.

Reply:
That error should fail automated tests too. We shouldn’t be relying on manual testing to find it.

Reply:
The Get Started button is clearing the active tutorial, which will crash the tutorial tab. I've removed `setActiveTutorial()` here:
https://github.com/zooniverse/front-end-monorepo/blob/928ef443ba9b2a1e700fa4a56239c6713e93e6b2/packages/lib-classifier/src/components/Classifier/components/TaskArea/TaskArea.js#L55-L58